### DISCUSSION TITLE
Sonoff Zigbee Dongle Plus E come router

### DISCUSSION THREAD

**Discussion by NuccioT** [indomus/forum#2477] [ZigBee, Z-Wave, Thread, Matter]
Buongiorno a tutti. Grazie alla guida inDomus sono riuscito a flashare due chiavette Sonoff Zigbee Dongle Plus E (V2) come router. La procedura indicata nella guida Sonoff (a cui rimanda la scheda inDomus) non è il massimo per chiarezza ma comunque alla fine si riesce.
La mia domanda ora è: il router lo devo accoppiare con il gateway e poi spostarlo oppure entra in funzione da solo?

**Comment by picardi21** [comment]
Ciao, come hai risolto? anche io ho lo stesso problema.
